support-frontend/assets/pages/paper-subscription-landing/helpers/windowHistoryUpdate.tsx

```diff
@@ -0,0 +1,12 @@
+export function windowHistoryUpdate(replace: string) {
```

Comment:
Because this is quite specific to updating the hash property I wonder if that should be made clear in the name? `windowSetHashProperty` or similar maybe?

Reply:
Yes that sounds a more accurate name, have gone with `windowSetHashProperty`